api/api-rules/violation_exceptions.list

```diff
@@ -57,6 +57,7 @@ API rule violation: names_match,k8s.io/apimachinery/pkg/runtime,Unknown,ContentT
 API rule violation: names_match,k8s.io/apimachinery/pkg/util/intstr,IntOrString,Type
 API rule violation: names_match,k8s.io/apimachinery/pkg/util/intstr,IntOrString,IntVal
 API rule violation: names_match,k8s.io/apimachinery/pkg/util/intstr,IntOrString,StrVal
+API rule violation: names_match,k8s.io/kube-proxy/config/v1alpha1,KubeProxyConfiguration,IPTables
```

Comment:
What does it mean for a config to be alpha? Are we saying its disabled by default and that it is allowed to change in backward incompatible ways between releases?

Reply:
Great point! I'll try to add this to my TODO list to document ;)
alpha means the API structure might change in incompatible ways and that usage of it is experimental (in some cases maybe even untested). Additionally the API version might be removed in favor for a newer API version in one minor release's timeframe.

Disabled by default doesn't really make sense because it's opt-in kinda automatically (the `--config` flag is optional and for some components even nonexistent)